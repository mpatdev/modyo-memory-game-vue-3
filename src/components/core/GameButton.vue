<script setup>

</script>

<template>
  <button class="game-button">
    <slot />
  </button>
</template>

<style lang="scss">
.game-button {
  @apply px-4 py-2 border border-black rounded-sm
}
</style>