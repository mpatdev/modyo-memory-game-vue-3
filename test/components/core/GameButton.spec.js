import GameButton from '@/components/core/GameButton.vue'

import snapshotTest from '@test/snapshotTest'

snapshotTest(GameButton, 'GameButton')