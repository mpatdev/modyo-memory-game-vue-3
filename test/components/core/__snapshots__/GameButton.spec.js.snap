// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GameButton > Check Snapshot 1`] = `"<button role=\\"button\\" class=\\"game-button\\"></button>"`;
